source/_integrations/slack.markdown

```diff
@@ -227,7 +227,7 @@ data:
           1.0
 ```
 
-Send a message directly to a user by setting the target to their member ID. Here are [instructions](https://www.workast.com/help/articles/61000165203/) to obtain a member ID.
+Send a message directly to a user by setting the target to their user ID. Here are [instructions](https://www.workast.com/help/articles/61000165203/) to obtain a member ID.
```

Comment:
```suggestion
Send a message directly to a user by setting the target to their user ID. Here are [instructions](https://www.workast.com/help/articles/61000165203/) to obtain a user ID.
```

Reply:
Thanks for the catch :)

Inconveniently Slack uses "Member ID" in their UI but "User ID" within their API docs. May be a moot point since this line will be removed by #30091 anyway.